.github/workflows/compat-check-latest-latest.yml

```diff
@@ -6,72 +6,37 @@ on:
   workflow_dispatch:
 
 jobs:
-  tests_lgpu_cpp:
-    name: Lightning Compatibility test (tests_lgpu_cpp) - latest/latest
-    uses: ./.github/workflows/lgpu_cpp.yml
-    with:
-      lightning-version: latest
-      pennylane-version: latest
-    secrets: inherit
-  tests_lgpumpi_cpp:
-    name: Lightning Compatibility test (tests_lgpumpi_cpp) - latest/latest
-    uses: ./.github/workflows/lgpumpi_cpp.yml
-    with:
-      lightning-version: latest
-      pennylane-version: latest
-    secrets: inherit
-  tests_linux_cpp:
-    name: Lightning Compatibility test (tests_linux_cpp) - latest/latest
-    uses: ./.github/workflows/linux_cpp.yml
-    with:
-      lightning-version: latest
-      pennylane-version: latest
-    secrets: inherit
-  tests_lkcuda_cpp:
-    name: Lightning Compatibility test (tests_lkcuda_cpp) - latest/latest
-    uses: ./.github/workflows/lkcuda_cpp.yml
-    with:
-      lightning-version: latest
-      pennylane-version: latest
-    secrets: inherit
-  tests_windows_cpp:
-    name: Lightning Compatibility test (tests_windows_cpp) - latest/latest
-    uses: ./.github/workflows/windows_cpp.yml
-    with:
-      lightning-version: latest
-      pennylane-version: latest
-    secrets: inherit
```

Comment:
Are those migrated elsewhere, or just not tested anymore here?
Would the CPP only tests be useful for Catalyst?

Reply:
I added them when I modified the files in #695  because I forgot we have been skipping them for the last few months. This new split makes the skip even more explicit.